tools/test.py

```diff
@@ -221,6 +221,8 @@ def main():
     cfg.model.train_cfg = None
     model = build_detector(cfg.model, test_cfg=cfg.get('test_cfg'))
     fp16_cfg = cfg.get('fp16', None)
+    if fp16_cfg is None and cfg.get('device', None) == 'npu':
+        fp16_cfg = dict(loss_scale=512.)
```

Comment:
same as above

Reply:
In MMCV, when using AMP in Torch, the loss_scale setting dynamic of FP16 only defaults to the initial value to 65536, and using a smaller as default initial value can avoid the incorrect update of the layer containing the buffer layer caused by excessive overflow at the beginning of training